src/ENSNotificationManager.m

```diff
@@ -452,7 +452,7 @@ - (NSString *)apiURL {
 }
 
 - (NSString *)appId {
-    return [[NSBundle mainBundle] objectForInfoDictionaryKey:@"ENSAppId"];
+    return (self.notificationAppId != nil) ? self.notificationAppId : [[NSBundle mainBundle] objectForInfoDictionaryKey:@"ENSAppId"];
```

Comment:
`return self.notificationAppId ?: [[NSBundle mainBundle] objectForInfoDictionaryKey:@"ENSAppId"];`

Reply:
ah i did not know the shorter syntax 👍